compiler/control/OMROptions.cpp

```diff
@@ -1163,7 +1162,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"traceLoopReplicator",              "L\ttrace loop replicator",                        TR::Options::traceOptimization, loopReplicator, 0, "P"},
    {"traceLoopStrider",                 "L\ttrace loop strider",                           TR::Options::traceOptimization, loopStrider,   0, "P"},
    {"traceLoopVersioner",               "L\ttrace loop versioner",                          TR::Options::traceOptimization, loopVersioner, 0, "P"},
-   {"traceMarkingOfHotFields",          "M\ttrace marking of Hot Fields",                 SET_OPTION_BIT(TR_TraceMarkingOfHotFields), "F"},
+   {"traceDynamicBreadthFirstHotFields", "M\ttrace marking of Hot Fields",                 SET_OPTION_BIT(TR_TraceDynamicBreadthFirstHotFields), "F"},
```

Comment:
Note this list is alphabetic - you must move the entry to the correct point in the list for the option to work. I'm not clear why we are changing the name of the option? To the JIT it is just reporting hot field data?

Reply:
Yes you are correct, but with the changes to remove the old manor of marking hot fields, anyone currently adding the “traceMarkingOfHotFields” option would not see anything reported unless “DynamicBreadthFirstScanOrdering” was also enabled. The goal here was just to add naming alignment. But as we discussed offline, we will leave the naming as it is and simply let the user know they will not see any hot fields without enabling DynamicBreadthFirstScanOrdering.